db/schema.rb

```diff
@@ -10,11 +10,10 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20170909082743) do
+ActiveRecord::Schema.define(version: 20170909175503) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
-  enable_extension "pg_stat_statements"
```

Comment:
This change seems unrelated. Can you either diagnose why your local postgres doesn't have this extension installed, or put this line back before committing the code?

Reply:
Thanks for your comment. The extension added to psql, and by a `rake db:migrate` it added back by rake.